src/Components/NotificationList/NotificationList.less

```diff
@@ -55,3 +55,7 @@
     color: #666;
     text-align: center;
 }
+
+.user{
+    width: 18%;

```

Comment:
You can't just add width to column. Because sum of all column width must be equal to `100%`. So you need to cut some percent from other columns